cli/pkg/local/app.go

```diff
@@ -24,7 +24,9 @@ import (
 	"go.uber.org/zap/zapcore"
 	"golang.org/x/sync/errgroup"
 
-	// blank import group.
+	runtimeserver "github.com/rilldata/rill/runtime/server"
+
```

Comment:
This isn't a blank import, just a renamed import – it should be grouped with the other 3rd party imports